includes/Plugins.php

```diff
@@ -0,0 +1,47 @@
+<?php
+namespace NewfoldLabs\WP\Module\AI;
+
+use NewfoldLabs\WP\Module\Installer\Services\PluginInstaller;
+
+/**
+ * Class Plugins
+ */
+final class Plugins {
+
+	/**
+	 * Install the plugin.
+	 *
+	 * @param string $plugin_slug the slug of the plugin that needs to be installed
+	 * @return boolean
+	 */
+	public static function install( $plugin_slug ) {
+		// the install function checks and installs the plugin
+		if ( ! PluginInstaller::install( $plugin_slug, true ) ) {
+			return false;
+		}
+		return true;
+	}
+
+	/**
+	 * Activate Jetpack modules.
+	 *
+	 * @param string $module the name of the module to activate
+	 *
+	 * @return boolean
+	 */
+	public static function activate_jetpack_module( $module ) {
```

Comment:
Does it make more sense for this to be in the installer or the onboarding module? The AI module's job should be to provide the patterns with the replaced content and images from AI, without worrying about how the consumer displays the block grammar to the user? Whether it works or not is also up to the consumer to handle?

I’m thinking that the code should reside in the installer, with the call to activate the module made in the onboarding or consumer module. Let me know what you think. We already have some code in the installer to prevent the WooCommerce activation redirect: https://github.com/newfold-labs/wp-module-installer/blob/2f7b1a2d128703119bc5090e93e9cdd3045044b4/includes/Data/Plugins.php#L224

Reply:
I agree that this could be placed in the `wp-module-installer` module so that it can be reused if needed. The only concern was that the decision to activate the module has to be taken only when a contact us page is being generated in the AI sitegen flow. Lets discuss.

Reply:
using `wp-module-installer` directly and moved the Jetpack module toggle function into that module